crates/bevy_ui/src/measurement.rs

```diff
@@ -50,33 +51,27 @@ impl Measure for FixedMeasure {
 pub struct ContentSize {
     /// The `Measure` used to compute the intrinsic size
     #[reflect(ignore)]
-    pub(crate) measure_func: Option<taffy::node::MeasureFunc>,
+    pub(crate) measure_func: Option<MeasureFunc>,
 }
 
 impl ContentSize {
     /// Set a `Measure` for this function
     pub fn set(&mut self, measure: impl Measure) {
-        let measure_func =
-            move |size: taffy::prelude::Size<Option<f32>>,
-                  available: taffy::prelude::Size<AvailableSpace>| {
-                let size =
-                    measure.measure(size.width, size.height, available.width, available.height);
-                taffy::prelude::Size {
-                    width: size.x,
-                    height: size.y,
-                }
-            };
-        self.measure_func = Some(taffy::node::MeasureFunc::Boxed(Box::new(measure_func)));
+        let measure_func = move |size: TaffySize<_>, available: TaffySize<_>| {
+            let size = measure.measure(size.width, size.height, available.width, available.height);
+            TaffySize {
+                width: size.x,
+                height: size.y,
+            }
+        };
+        self.measure_func = Some(MeasureFunc::Boxed(Box::new(measure_func)));
```

Comment:
This looks much cleaner. We should be using these type aliases in the `layout` module too.